test/e2e/e2e_test.go

```diff
@@ -1392,6 +1392,24 @@ func waitForRCPodsRunning(ctx context.Context, t *testing.T, clientSet clientset
 	}
 }
 
+func waitForTerminatingPodsToDisappear(ctx context.Context, t *testing.T, clientSet clientset.Interface, namespace string) {
```

Comment:
Here you are putting back `waitForTerminatingPodsToDisappear` which was removed in https://github.com/kubernetes-sigs/descheduler/pull/1500/commits/1e23dbcc2669f69288ec6c9dbcd51fa7b3c1c8e5. Can you consolidate the changes into a single commit?